src/Symfony/Bridge/Doctrine/Tests/Form/Type/EntityTypeTest.php

```diff
@@ -1366,6 +1366,180 @@ public function testDefaultTranslationDomain()
         $this->assertNull($view['child']->vars['translation_domain']);
     }
 
+    public function testPassLabelTranslationParametersToView()
```

Comment:
`testPassLabelTranslationParametersToView` is overriden in `EntityTypeTest` because it requires additional parameters :

```php
'em' => 'default',
'class' => self::SINGLE_IDENT_CLASS,
```
Otherwise it will throw an exception.